src/docs/product/performance/transaction-summary.mdx

```diff
@@ -74,12 +74,33 @@ The sidebar contains helpful supplementary information about this transaction's
 
 Frontend transactions will have a "Web Vitals" tab. Clicking on the tab will take you to the [Web Vitals page](/product/performance/web-vitals), where you can see a detailed view of the web vitals associated with this transaction.
 
+## Tags
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+On this tab, suspect tag keys that often correspond to slower transactions are listed. Toggle through them to have corresponding tag values reflected in the heat map. Similar to a histogram, events are distributed by duration. Click on any purple box to view a list of events or continue to filter down the events by targeting values in the table below the heat map.
+
+![Example of Tags Tab](./tags-tab.png)
+
+## Events
```

Comment:
Assuming the name of this tab isn't going to change, all of the other headings for tabs are the same as the name of the tab.

```suggestion
## All Events
```